src/models/People.js

```diff
@@ -69,11 +69,7 @@ static async displayPeopleProfessional() { // returns array of Timesheet Table
       LEFT JOIN capabilities ON capabilities.id = group_capabilities.capabilities_id
       WHERE people_groups.people_id = :people_id AND capabilities.name IS NOT NULL`;
 
-      const rolesSQL = `SELECT DISTINCT groups.name from people_groups 
-      LEFT JOIN groups ON groups.id = people_groups.group_id
-      LEFT JOIN group_capabilities ON group_capabilities.group_id = groups.id
-      LEFT JOIN capabilities ON capabilities.id = group_capabilities.capabilities_id
-      WHERE people_groups.people_id = :people_id AND capabilities.name IS NOT NULL`;
+      const rolesSQL = `SELECT groups.name FROM groups LEFT JOIN people_groups ON groups.id = people_groups.group_id WHERE people_groups.people_id = :people_id`;
```

Comment:
Why are we removing the capabilities?

Our permissions system revolves around capabilities, not groups.

Reply:
The purpose of this SQL statement is to just find the groups/roles of a user (SELECT groups.name FROM). I just merely shortened it, and included all of the groups that don't have capabilities :). Since roles are deprecated, I can rename this to groups.

The code for finding the capabilities is directly above this line of code (capabilitiesSQL, not written by me).

Reply:
Ah okay, got it. I agree with your change. 👍

Actually, we can put a "deprecated" mark here if we are deprecating the idea of `roles`.